.github/workflows/wiki.yaml

```diff
@@ -0,0 +1,23 @@
+name: Documentation on the wiki page
```

Comment:
Let's do this on Taskcluster since that's where the rest of the CI is running (https://github.com/mozilla/relman-auto-nag/blob/master/.taskcluster.yml).

Reply:
Or let's use a GitHub Action, in the end it's the same :)